### ISSUE TITLE
./qemu.sh

### ISSUE THREAD

**Issue by TreeBial** [swsnu/osspr2021#11]
프로젝트 수행을 위해 몇 가지를 수행하고 나니
./qemu.sh 명령어에서 다음과 같은 오류가 나오는데
무엇이 문제인가요...?
![image](https://user-images.githubusercontent.com/51820025/110595771-cc3aaa00-81c1-11eb-99bb-3a508e8351ee.png)

**Comment by ooffordable** [comment]
네, 해당 에러는 `qemu-system-aarch64`가 제대로 설치되지 않았거나, 현재 시스템이  qemu의 build PATH를 알고있지 못해서 발생하는 에러로 추측됩니다. 
만약 `qemu-system-aarch64` 설치까지 완료하셨다면, @pullup-dip-burpee 님이 알려주신대로, PATH를 추가해보시길 바랍니다. 
감사합니다.

**Comment by TreeBial** [comment]
답변 감사합니다!
덕분에 문제 해결했습니다.
